template/README.template.md

```diff
@@ -0,0 +1,22 @@
+# Server Folder
+
+This folder contains the server code for the project.
+It contains the following files:
+
+- `bin/server.dart` - The main entry point for the server.
+- `bin/middlewares.dart` - Contains all the middlewares for the server. Contains all CSP, CORS, and other security headers.
+- `Dockerfile` - The Dockerfile for the server to build it.
+
+## Running the server
+
+The recommended way to run the server is to use the Dockerfile by using the following command:
```

Comment:
Maybe `The recommended way to run the server is to run it within docker with the following command:` 🤔

I think for people not familiar with docker the distinction between the actual server and docker is not clear and we should try to make it more clear 🤔

Reply:
Done